tests/unit-core/test-api-objecttype.c

```diff
@@ -121,6 +122,16 @@ main (void)
     jerry_release_value (entries[idx].value);
   }
 
+  /* Module has no feature detector. */
```

Comment:
Good to know, this should be added later.

Reply:
Added now.